Makefile

```diff
@@ -457,17 +460,26 @@ endif
 wasmtest:
 	$(GO) test ./tests/wasm
 
-build/release: tinygo gen-device wasi-libc
+build/compiler-rt.%.a: build/tinygo$(EXE)
+	./build/tinygo$(EXE) build-library -target=$* -o $@ compiler-rt
```

Comment:
Normally we use `$(TINYGO)` for this. Would that also work?

Reply:
Ah, I wrote this long ago and rebased; probably you've changed how this is done a bit. That'll likely work too.

Reply:
Actually, the [replaced lines](https://github.com/tinygo-org/tinygo/pull/2091/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L491-L496) are using the built version, so it seems like this should too?

Reply:
Right, I see. That makes sense. It looks good to me as-is.